fastlane/Fastfile

```diff
@@ -914,7 +914,7 @@ inal copy, and try again.")
   lane :symbols_upload do |options|
     symbols_path = options[:dsym_path] || lane_context[SharedValues::DSYM_OUTPUT_PATH]
     sentry_upload_dsym(
-      auth_token: get_required_env('SENTRY_AUTH_TOKEN'),
+      auth_token: ENV.fetch('SENTRY_AUTH_TOKEN'),
```

Comment:
@chipsnyder FYI: we don't have `get_required_env` on our `Fastfile`, so I had to fix that. :)